books/english-composition/bake

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '9.0.0'
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :english_composition) do |doc|
+  include Kitchen::Directions
+
+  # # Set overrides
+  # doc.selectors.override(
+  #   page_summary: 'section.section-summary',
+  #   reference: 'section.references'
+  # )
+
+  book = doc.book
+  metadata = book.metadata
+
+  # Some stuff just goes away
+  book.search('cnx-pi').trash
+
+  BakePreface.v1(book: book)
+  BakeHandbook.v1(book: book)
+
+  # # Bake NumberedTable in Preface
+  # book.pages('$.preface').tables('$:not(.unnumbered)').each do |table|
+  #   BakeNumberedTable.v1(table: table, number: "#{table.count_in(:page)}")
+  # end
+
+  BakeChapterTitle.v1(book: book)
+  BakeChapterIntroductions.v1(book: book)
+  BakeUnitTitle.v1(book: book)
+
+  # Bake EoC sections
+  book.chapters.each do |chapter|
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'further-reading',
+      uuid_key: '.further-reading',
+      section_selector: 'section.further-reading',
+      include_intro_page: false
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'works-cited',
+      uuid_key: '.works-cited',
+      section_selector: 'section.works-cited',
+      include_intro_page: false
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
+    MoveCustomSectionToEocContainer.v1(
+      chapter: chapter,
+      metadata_source: metadata,
+      container_key: 'works-consulted',
+      uuid_key: '.works-consulted',
+      section_selector: 'section.works-consulted',
+      include_intro_page: false
+    ) do |section|
+      RemoveSectionTitle.v1(section: section)
+    end
```

Comment:
ah, I see why you asked about passing multiple sections to this direction! another way to make this shorter (if the class names are the only difference in all of these) would be to iterate over an array of the class names and call the direction within the for-loop

Reply:
Yes, class is the only difference as far as I remember. I was wondering if we could rework `MoveCustomSectionToEocContainer` similarly as we did to `BakeAutotitledNotes` which can utilize many classes or just one if that's the case.

Reply:
As Kendra suggested I added the loop containing classes outside `MoveCustomSectionToEocContainer`. because all of the sections need the same settings.